LiteCore/RevTrees/RevTree.hh

```diff
@@ -67,11 +67,29 @@ namespace litecore {
     private:
         slice       _body;          /**< Revision body (JSON), or empty if not stored in this tree*/
 
+        enum DataFlags : uint8_t {
```

Comment:
It would be simpler just to add two bool fields, `_hasInsertedRevID` and `_hasInsertedBody`. Then you can get and set them directly instead of creating a bunch of accessors. If you're really concerned about the extra byte you could make them bitfields, but I think alignment is going to cause the struct to grow by more than 2 bytes no matter what you add.

Reply:
Done